src/storage/media.js

```diff
@@ -0,0 +1,33 @@
+const config = require('../config');
+
+const mediaTypes = {
+  EVENT_COVER: 1,
+  PROFILE_PICTURE: 2,
+};
+
+const getMediaTypeConfig = (mediaType) => {
+  let mediaTypeConfig;
+  switch (mediaType) {
+    case mediaTypes.EVENT_COVER:
+      mediaTypeConfig = {
+        type: mediaTypes.EVENT_COVER,
+        maxFileSize: config.file.MAX_EVENT_COVER_FILE_SIZE,
+        uploadPath: config.file.EVENT_COVER_UPLOAD_PATH,
+        fileTag: 'image',
+      };
+      break;
+    case mediaTypes.PROFILE_PICTURE:
+      mediaTypeConfig = {
+        type: mediaTypes.PROFILE_PICTURE,
+        maxFileSize: config.file.MAX_PROFILE_PICTURE_FILE_SIZE,
+        uploadPath: config.file.EVENT_COVER_UPLOAD_PATH,
+        fileTag: 'image',
+      };
+      break;
+    default:
+      mediaTypeConfig = {};
```

Comment:
Throw an error here.

This pattern matching should be exhaustive, and if it's not, that's something we want to point out immediately instead of potentially failing silently, especially because it'll be an error caused by adding a new media type and not properly configuring it. Unfortunately, I don't think TypeScript does exhaustiveness checks like other languages do.